pkg/operator/ceph/config/defaults.go

```diff
@@ -71,33 +74,37 @@ func DefaultCentralizedConfigs(cephVersion version.CephVersion) *Config {
 	if !cephVersion.IsAtLeast(version.CephVersion{Major: 14, Minor: 2, Extra: 1}) {
 		// Set the default log files to empty so they don't bloat containers. Can be changed in
 		// Mimic+ by users if needed.
-		c.Section("global").
-			Set("log file", "").
-			Set("mon cluster log file", "")
+		overrides = append(overrides, []rookceph.ConfigOverride{
+			configOverride("global", "log file", ""),
+			configOverride("global", "mon cluster log file", ""),
+		}...)
 	}
 
-	return c
+	return overrides
 }
 
 // DefaultLegacyConfigs need to be added to the Ceph config file until the integration tests can be
 // made to override these options for the Ceph clusters it creates.
-func DefaultLegacyConfigs() *Config {
-	c := NewConfig()
-	c.Section("global").
-		Set("mon max pg per osd", "1000").
+func DefaultLegacyConfigs() rookceph.ConfigOverridesSpec {
+	overrides := []rookceph.ConfigOverride{
+		configOverride("global", "mon max pg per osd", "1000"),
 		//
-		Set("osd pg bits", "11").
-		Set("osd pgp bits", "11").
-		Set("osd pool default size", "1").
-		Set("osd pool default min size", "1").
-		Set("osd pool default pg num", "100").
-		Set("osd pool default pgp num", "100").
+		// configOverride("global", "osd pg bits", "11"), // not available in nautilus
+		// configOverride("global", "osd pgp bits", "11"), // not available in nautilus
+		configOverride("global", "osd pool default size", "1"),
+		configOverride("global", "osd pool default min size", "1"),
+		configOverride("global", "osd pool default pg num", "100"),
+		configOverride("global", "osd pool default pgp num", "100"),
+		//
+		// TODO: Is this still necessary?
+		configOverride("global", "rbd_default_features", "3"),
 		//
-		Set("rbd_default_features", "3"). // TODO: still needed?
 		// Setting fatal signal handlers to true (the default) will print a lot of extra information
 		// from daemons when they encounter a failures, but it is VERY verbose. When the mon kv
 		// store is available, it will probably be best to set this to false by default if it is
 		// unset, but leave it set to true if the user has specified it as true during runtime.
-		Set("fatal signal handlers", "false")
-	return c
+		// configOverride("global", "fatal signal handlers", "false"),
+		// error: Configuration option 'fatal_signal_handlers' may not be modified at runtime
```

Comment:
Why not set it anyways? Since the set flag as soon as possible, only the first mon won't have this option enabled and it'll need to be restarted to read that value and apply it. However, other daemons starting for the first time (mon.b, mon.c and the rest) will benefit from that flag right away.

Reply:
The logs spew out the error every time `ceph config set` is used. It's pretty unnerving, and the message coming back from isn't clear that that is the behavior or why it's printing the error.

Reply:
Also, I'm not sure that suppressing `fatal_signal_handlers` is necessarily a good idea. It's more verbose, but only in failure conditions where that verbosity might be useful. @travisn & @galexrt how do you feel about that?